tests/TestCase.php

```diff
@@ -1,16 +1,27 @@
 <?php
 
+use Orchestra\Testbench\Foundation\PackageManifest;
+use Orchestra\Testbench\TestCase as TestbenchTestCase;
 use PHPUnit\Framework\Assert;
+use Winter\Storm\Foundation\Application;
 
-class TestCase extends PHPUnit\Framework\TestCase
+class TestCase extends TestbenchTestCase
 {
     /**
-     * Creates the application.
+     * Resolve application implementation.
      *
-     * @return Symfony\Component\HttpKernel\HttpKernelInterface
+     * @return \Winter\Storm\Foundation\Application
      */
-    public function createApplication()
+    protected function resolveApplication()
     {
+        return tap(new Application($this->getBasePath()), function ($app) {
+            $app->bind(
+                'Winter\Storm\Foundation\Bootstrap\LoadConfiguration',
+                'Orchestra\Testbench\Bootstrap\LoadConfiguration'
```

Comment:
For class strings, should we use the `::class` constant instead for better IDE integration?